apps/api/src/utils/express/corsConfig.ts

```diff
@@ -1,18 +1,7 @@
 import type cors from 'cors';
 
 const corsConfig: cors.CorsOptions = {
-  origin: (origin, callback) => {
-    if (
-      !origin ||
-      origin.includes('hey.xyz') ||
-      origin.includes('heyxyz.vercel.app') ||
-      origin.startsWith('http://localhost')
-    ) {
-      callback(null, true);
-    } else {
-      callback(new Error('Not allowed by CORS'));
-    }
-  }
+  origin: ['hey.xyz', /\.hey\.xyz$/, /-heyxyz\.vercel\.app$/]
```

Comment:
does it allow localhost?

Reply:
it's not necessary. localhost:4783 is the API's startup port, and there is no cross-origin issue when accessing it from the same domain.

Reply:
In general, when debugging locally, cross-origin issues can be resolved by using a proxy to make the requests.